### ISSUE TITLE
Admin HTTP API to register demo Operations & Orchestrations

### ISSUE THREAD

**Issue by pedrorgirardi** [DEX-Company/surfer#127]

**Comment by mikera** [comment]
Good idea! We might want to consider a few related config options, some thoughts:

- We could potentially allow registered users to upload and run any orchestration, without pre-registration. There is reduced risk from this, since they still need access to the underlying operations / data through separate means
- I think we should force orchestrations to have (and validate) a content-hash by default. without this there is a risk of DOS attacks through self-reference (i.e. an orchestration could contain a call to itself)
- Operations which are implemented in Clojure / surfer code definitely need to be explicitly enabled, though it might be possible to pre-register a set of default / test operations (will be useful for demos etc.)
